contractcourt/chain_watcher.go

```diff
@@ -465,53 +468,18 @@ func (c *chainWatcher) dispatchCooperativeClose(commitSpend *chainntnfs.SpendDet
 		SettledBalance: localAmt,
 		CloseType:      channeldb.CooperativeClose,
 		ShortChanID:    c.cfg.chanState.ShortChanID(),
-		IsPending:      true,
+		IsPending:      false,
```

Comment:
Why false? It's fully closed here as this will only be dispatched once the co-op close transaction hits the chain.

Reply:
`!IsPending` means that it is fully closed.